ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,5 @@
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
 <launch>
-  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen"/>
+  <arg name="ID" default="$(optenv ID 1)"/>
```

Comment:
For the argument name use `id` instead of `ID`.

Reply:
Adressed